google/cloud/aiplatform/metadata/schema/utils.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from typing import Optional, Dict, List
+from dataclasses import dataclass
+
+
+@dataclass
+class PredictSchemata:
```

Comment:
Why not alias to the protos here? It would also keep them in sync.

Reply:
We also discussed about this in our team meeting. There is two reasons we decide to not refer to the proto here. First is that the underline source of truth are different, so one could change the proto def or our type def in the future and they would not be in sync by design anymore, and referring to the proto in the SDK would be invalid by then. Second is that we want to make the schema def independent of platform as they might be used by KFP in the future, so the link to a google specific proto could be an issue at that point.

Reply:
per offline discussion resolving as wont fix.